osu.Game/Graphics/UserInterface/PerformanceCounter.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Allocation;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class PerformanceCounter : RollingCounter<double>
+    {
+        protected override double RollingDuration => 250;
+
+        public PerformanceCounter()
+        {
+            Current.Value = DisplayedCount = 0;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours) => AccentColour = colours.BlueLighter;
+
+        protected override string FormatCount(double count)
+        {
+            return $@"{Math.Round(count, 2)}pp";
```

Comment:
Minor nitpick, but you can use format specifiers here.
```suggestion
            return $@"{count:N2}pp";
```